yt/frontends/ramses/fields.py

```diff
@@ -104,7 +110,7 @@ class RAMSESFieldInfo(FieldInfoContainer):
         ("particle_velocity_z", (vel_units, [], None)),
         ("particle_mass", ("code_mass", [], None)),
         ("particle_identifier", ("", ["particle_index"], None)),
-        ("particle_age", ("code_time", ['age'], None)),
+        ("particle_formation_time", ("code_time", ['age'], None)),
```

Comment:
To be coherent with above, maybe we should also define the `conformal_formation_time` for sinks?

Reply:
I actually wanted to check with you about this. For sinks the on-disk field means exactly the same thing as for regular particles? That is, for cosmology simulations it's a conformal time when the particle formed and for non-cosmological it's the physical time the particle formed in code units?

Reply:
The time for the sinks is given in the same units as the one for any particle (so conformal for cosmological simulation, else physical time in code units). Maybe @HugoPfister can confirm that.